package.json

```diff
@@ -7,9 +7,10 @@
     "lerna-changelog": "cli.js"
   },
   "scripts": {
-    "build": "babel src -d lib",
+    "build": "npm run clean && babel src --out-dir lib --ignore src/__mocks__/GithubAPI.js,src/__mocks__/Changelog.js",
```

Comment:
I also found [this thread](https://github.com/facebook/jest/issues/2070#issuecomment-265027510) from you @cpojer where you mention

```
Manual mocking is really messed up and doesn't work well.
```

Do you still recommend to use them or to "wait until they're fixed"?

Besides the publishing part, what about compiled sources from babel? Should we manually ignore those or is there a better way of doing it?
Thanks for the help!